rotkehlchen/chain/evm/manager.py

```diff
@@ -51,6 +63,135 @@ def get_historical_balance(
         returns None"""
         return self.node_inquirer.get_historical_balance(address, block_number)
 
+    def query_balances(
+            self,
+            addresses: Sequence[ChecksumEvmAddress],
+    ) -> dict[ChecksumEvmAddress, BalanceSheet]:
+        return self.query_protocols_with_balance(
+            balances=self.query_evm_chain_balances(accounts=addresses),
+        )
```

Comment:
ah I didn't notice that here you query both the balance + tokens + protocols. Can you add a docstring to the function explaining what you query and the possible errors?